lib/util/ajv.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @fileoverview The instance of Ajv validator.
+ * @author Evgeny Poberezkin
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const Ajv = require("ajv"),
+    metaSchema = require("ajv/lib/refs/json-schema-draft-04.json");
+
+//------------------------------------------------------------------------------
+// Public Interface
+//------------------------------------------------------------------------------
+
+const ajv = new Ajv({
+    meta: false,
+    validateSchema: false,
+    verbose: true
+});
+
+ajv.addMetaSchema(metaSchema);
+/* eslint no-underscore-dangle: "off" */
```

Comment:
I believe this will disable the rule for the whole file. I think you might want to use `// eslint-disable-next-line no-underscore-dangle`instead?